[Docs] Threat score follow up changes (#20953)
cloudflare/cloudflare-docs: src/content/partials/waf/threat-score-definition.mdx

## Threat score

> Previously, a threat score represented a Cloudflare threat score from 0–100, where 0 indicates low risk. Now, a threat score is default to 0.

Reviewer: ```suggestion
Previously, a threat score represented a Cloudflare threat score from 0–100, where 0 indicates low risk. Now, the threat score is always `0` (zero).
```

Reviewer: You should also update the description in the field reference:
https://github.com/cloudflare/cloudflare-docs/blob/production/src/content/fields/index.yaml#L606
